### PR TITLE
Added AddMemory,MyPeople,SearchPage component

### PR DESCRIPTION
**Pull request by rohit-kumar6**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -20,6 +23,9 @@ function App() {
             <Route path="/login" component={LoginPage}></Route>
             <Route path="/register" component={RegisterPage}></Route>
             <Route path="/about" component={AboutPage}></Route>
+            <Route path="/add" component={AddMemoryPage}></Route>
+            <Route path="/myPeople" component={MyPeoplePage}></Route>
```
[the remark this answers is not in the document]
Comment (NitronR) [reply src/App.js]:
```suggestion
            <Route path="/my-people" component={MyPeoplePage}></Route>
```

File: src/component/404/404.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Card from "react-bootstrap/Card";
+import Button from "react-bootstrap/Button";
+import { Link } from "react-router-dom";
+import "./404.css";
+
+function AboutPage() {
```
[the remark this answers is not in the document]
Comment (rohit-kumar6) [reply src/component/404/404.jsx]: Done

File: src/component/404/404.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  return (
+    <Card>  
+      <h3>Page Not found</h3>
+      <center><Link to="/">Return to Home Page</Link></center>
+    </Card>
+  );
+}
+
+export default AboutPage;
```
[the remark this answers is not in the document]
Comment (rohit-kumar6) [reply src/component/404/404.jsx]: Done

File: src/App.js [hunk 4]
```diff
@@ -20,6 +23,9 @@ function App() {
             <Route path="/login" component={LoginPage}></Route>
             <Route path="/register" component={RegisterPage}></Route>
             <Route path="/about" component={AboutPage}></Route>
+            <Route path="/add" component={AddMemoryPage}></Route>
```
[the remark this answers is not in the document]
Comment (rohit-kumar6) [reply src/App.js]: Done

File: src/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (rohit-kumar6) [reply src/App.js]: Done

File: src/component/AddMemoryPage/AddMemoryPage.jsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Card from "react-bootstrap/Card";
+import Button from "react-bootstrap/Button";
```
[the remark this answers is not in the document]
Comment (rohit-kumar6) [reply src/component/AddMemoryPage/AddMemoryPage.jsx]: Done

File: src/component/MyPeoplePage/MyPeoplePage.jsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Card from "react-bootstrap/Card";
+import Button from "react-bootstrap/Button";
```
[the remark this answers is not in the document]
Comment (rohit-kumar6) [reply src/component/MyPeoplePage/MyPeoplePage.jsx]: Done

**Pull request merged**